src/pages/home/report/ReportActionItem.js

```diff
@@ -162,44 +192,40 @@ class ReportActionItem extends Component {
                     event.target.blur();
                 }}
             >
-                <Hoverable resetsOnClickOutside>
-                    {hovered => (
-                        <View>
-                            {this.props.shouldDisplayNewIndicator && (
-                                <UnreadActionIndicator />
-                            )}
-                            <View
-                                style={StyleUtils.getReportActionItemStyle(
-                                    hovered
-                                    || this.state.isContextMenuActive
-                                    || this.props.draftMessage,
-                                )}
-                            >
-                                {!this.props.displayAsGroup
-                                    ? (
-                                        <ReportActionItemSingle action={this.props.action} showHeader={!this.props.draftMessage}>
-                                            {children}
-                                        </ReportActionItemSingle>
-                                    )
-                                    : (
-                                        <ReportActionItemGrouped>
-                                            {children}
-                                        </ReportActionItemGrouped>
-                                    )}
-                            </View>
-                            <MiniReportActionContextMenu
-                                reportID={this.props.reportID}
-                                reportAction={this.props.action}
-                                displayAsGroup={this.props.displayAsGroup}
-                                isVisible={
-                                    hovered
-                                    && !this.state.isContextMenuActive
-                                    && !this.props.draftMessage
-                                }
-                                draftMessage={this.props.draftMessage}
-                            />
-                        </View>
+                <Hoverable resetsOnClickOutside onHoverIn={this.setHoveredWhenNotScrolling} onHoverOut={this.unsetHovered}>
```

Comment:
**thought:** A better design might be to have this be the responsibility of the `Hoverable` if we don't want things to hover on scroll then have a scroll listener in `Hoverable` and don't pass the hover state until after we stop scrolling.

Reply:
Good idea, I can make a wrapper around the Hoverable. But we can't modify `Hoverable` as it is generic and other parts of the app do not depend on report scrolling. 

Question: How will I listen to the scroll event of the report messages list? 

I can register a callback in the ReportScrollManager but there will be many Hoverable items on the Action list. It seems complicated to manage those. Not getting any other ideas.

Reply:
Hmm. Good point. Why I didn't think of this? :smile: